app/views/provider_interface/provider_relationship_permissions/edit.html.erb

```diff
@@ -1,4 +1,5 @@
 <% content_for :browser_title, title_with_error_prefix('', @form.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(provider_interface_organisation_path(@form.training_provider)) %>
```

Comment:
Hopefully this is fine, as I don't think you could go from the ratifying provider page to the edit permissions page?

Reply:
Oh, I suppose I can also just not specify where that back link goes